<launch>
  <include file="$(find fetch_vr)/launch/aruco_marker_publisher.launch" />

  <node name="moveit_planning_scene_publisher" pkg="fetch_vr" type="moveit_planning_scene_publisher.py">
    <param name="failure_duration" value="10.0" />
    <param name="marker_frame" value="marker_200" />
    <param name="reference_frame" value="base_link" />
  </node>
</launch>
